### PR TITLE
Added some transcript corrections

### PR DESCRIPTION
**Pull request by chrisn**
This PR makes some corrections to the workshop transcripts. There's a lot remaining that could be improved, but I've done what I can.

### CODE REVIEW

File: session-1.html [hunk 1]
```diff
@@ -823,40 +822,40 @@ <h3>Media Synchronization</h3>
...
+            <cite>Chris Needham:</cite> Does the output latency only apply to the AudioContext, or does it also apply to the audio or video element?
           </p>
 
           <p>
-            <cite>Paul Adenot:</cite> This particular audio context, and the audio frames sent to the output. The media elements clock include audio latency offsets built in because the purpose is not the same.
-            <br/>... If the current time is exactly 1.00 seconds, it is expected that you hear the sound, it is 1.00 second. The video frames that have been offset for you, that's much higher level constructs.
-            <br/>... There is also the case of the element into the audio context. That's also different. You have to offset at this point.
+            <cite>Paul Adenot:</cite> This particular for the AudioContext. The media elements' clock includes audio latency ofsets built in because the purpose is not the same.
+            <br/>... If the currentTime is exactly 1.00 seconds, it is expected that you hear the soundat that same time, 1.00 seconds. The video frames that have been offset for you, that's a much higher level construct.
```
Comment (tidoust) [inline session-1.html:850]:
```suggestion
            <br/>... If the currentTime is exactly 1.00 seconds, it is expected that you hear the sound at that same time, 1.00 seconds. The video frames that have been offset for you, that's a much higher level construct.
```

File: session-2.html [hunk 2]
```diff
@@ -201,31 +201,31 @@ <h4>Advanced needs: realtime captioning, alpha video</h4>
             </p>
 
             <p>
-              <cite>Pierre-Anthony Lemieux:</cite> Multichannel audio. Higher quality video. Any other requests? 
+              <cite>Pierre-Anthony Lemieux:</cite> Multichannel audio. Higher quality video. Any other requests?
             </p>
 
             <p>
-              <cite>Sergio Garcia Murillo:</cite> The topics I highlighted in the presentation, for example, is support for Web Realtime Subtitling. To be able to provide in realtime.
+              <cite>Sergio Garcia Murillo:</cite> The topics I highlighted in the presentation, for example, is support for real time sSbtitling on the web.
```
Comment (tidoust) [inline session-2.html:208]:
```suggestion
              <cite>Sergio Garcia Murillo:</cite> The topics I highlighted in the presentation, for example, is support for real time subtitling on the web.
```

File: session-2.html [hunk 3]
```diff
@@ -237,13 +237,13 @@ <h4>Advanced needs: realtime captioning, alpha video</h4>
...
-              <br/>... An API for being able to control the behavior of the jitter buffer and provide a hint to the implementation that what you want to play is music. And apply a different policy to not do any stretching of the audio, distortions. Just apply a different policy for packet loss. 
+              <cite>Sergio Garcia Murillo:</cite> I think one other thing that is both an implementation and a spec issue is being able to surface an API control to say whether the audio playback is for music. Specifically, implementations of the jitter buffer may change the bits and disturb the audio. It's okay if you are doing a voice conference or things like that. But for music, it could cause problems.
+              <br/>... An API for being able to control the behavior of the jitter buffer and provide a hint to the implementation that what you want to play is music. And apply a different policy to not do any kind of stretching of the audio, which causs distortions. Just apply a different policy for packet loss.
```
Comment (tidoust) [inline session-2.html:241]:
```suggestion
              <br/>... An API for being able to control the behavior of the jitter buffer and provide a hint to the implementation that what you want to play is music. And apply a different policy to not do any kind of stretching of the audio, which causes distortions. Just apply a different policy for packet loss.
```

**Review by tidoust — APPROVED** [review]
Many thanks! Just a few typos to fix on the typo fixes...

**Comment by chrisn** [comment]
Thank you!

**Pull request merged**